### PR TITLE
Add mute/unmute chronicles events.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: src/components/AVPlayer/AVPlayer.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ class AVPlayer extends Component {
     const { item } = this.props;
     if (!isEqual(this.state.item, item)) {
       if (this.isUnitExistAndPlaying()) {
-        this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props.media));
+        this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props));
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:209]: Please look three lines above: there is `item` already. Please take `src` from `this.state` too. Why do you need the whole `this.props`? To take something from it? Or take it here, or take items when already inside.

File: src/components/AVPlayer/AVPlayer.js [hunk 2]
```diff
@@ -225,7 +225,7 @@ class AVPlayer extends Component {
     }
     window.removeEventListener('message', this.receiveMessageFunc, false);
     if (this.isUnitExistAndPlaying()) {
-      this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props.media));
+      this.props.chronicles.append('player-stop', this.buildAppendData(this.props.item, this.state.src, this.props));
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:228]: Same questions as above.

File: src/components/AVPlayer/AVPlayer.js [hunk 3]
```diff
@@ -359,12 +359,16 @@ class AVPlayer extends Component {
...
   };
 
-  buildAppendData = (item, src, media) => {
+  buildAppendData = (item, src, props) => {
     return {
       unit_uid: item.unit.id,
       file_src: src,
-      current_time: media.currentTime,
-      duration: media.duration,
+      current_time: props.media.currentTime,
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:366]: If you need media, then call this function with props.media. And inside this function use destructuring.

File: src/components/AVPlayer/AVPlayer.js [hunk 4]
```diff
@@ -376,7 +380,7 @@ class AVPlayer extends Component {
     const unitId = item?.unit?.id;
     if (unitId) {
       const {unit, item: { mediaType }, selectedLanguage, uiLanguage } = this.props;
-      this.props.chronicles.append('player-play', this.buildAppendData(item, this.state.src, this.props.media));
+      this.props.chronicles.append('player-play', this.buildAppendData(item, this.state.src, this.props));
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:383]: maybe somewhere at the beginning to write `{ chronicles } = this.props` and to use it everywhere? Just remember that javascript is interpreted language.

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/AVPlayer/AVPlayer.js (hunk 1 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:209]: Good point, taking all params now from this.

File: src/components/AVPlayer/AVPlayer.js (hunk 2 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:228]: Good point, taking all params now from this.

File: src/components/AVPlayer/AVPlayer.js (hunk 3 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:366]: Taking all params now from this.

File: src/components/AVPlayer/AVPlayer.js (hunk 4 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:383]: Done

**Comment by bbfsdev** [comment]
Finished all fixes, thanks!

**Pull request merged**